src/main/java/org/mockito/invocation/Invocation.java

```diff
@@ -47,6 +50,13 @@
      */
     Object[] getRawArguments();
 
+    /**
+     * Returns arguments wrapped into ArgumentMatchers
+     *
```

Comment:
Can you add a note that the method converts each argument using ArgumentMatchers.eq() or AdditionalMatchers.arryEq() and that it is used internally for the purposes of human-readable invocation printing.

I don't want users to confuse this new method with the MatchableInvocation.getMatchers() method.